install/post_install.sh

```diff
@@ -5,7 +5,7 @@ SYSTEMD="/lib/systemd/system"
 UPSTART="/etc/init"
 SYSV1="/etc/init.d"
 SYSV2="/etc/rc.d/init.d/"
-DIR="/opt/tyk-gateway/install"
+DIR="/opt/tyk/install"
```

Comment:
If not required let's avoid install directory changes please as it might potentially break existing users/cause confusion.